jupyter_elfinder/elfinder.py

```diff
@@ -543,39 +537,34 @@ def __zipdl(self) -> None:
         pass
 
     def __file(self) -> None:
+        self._http_header["Content-type"] = "text/html"
```

Comment:
Content type header is added here since it's the same for all cases in this command.